Source/WebInspectorUI/UserInterface/Models/DOMNodeStyles.js

```diff
@@ -362,7 +362,7 @@ WI.DOMNodeStyles = class DOMNodeStyles extends WI.Object
     {
         selector = selector || this._node.appropriateSelectorFor(true);
 
-        let result = new WI.WrappedPromise;
+        let promise = Promise.withResolvers();
```

Comment:
Isn't it a bit confusing to call this `promise` given that you get `{ promise, resolve, reject }`?

Reply:
yeah i think it'd be fine to keep it as `result` as then the other lines dont need to change